.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v[0-9]+(\.[0-9]+)*'
+jobs:
+  publish:
+    name: Publish to npm
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '14' ]
```

Comment:
Let's go with node 16. It looks like [it's "current"](https://nodejs.org/en/about/releases/).